intellij-plugin/src/main/kotlin/io/github/ethersync/protocol/CursorEvent.kt

```diff
@@ -0,0 +1,15 @@
+package io.github.ethersync.protocol
+
+import com.google.gson.annotations.SerializedName
+import org.eclipse.lsp4j.Range
+
+data class CursorEvent(
+   @SerializedName("userid")
+   val userId: String,
```

Comment:
The editor dev guide says

> #### `"cursor" {userid: integer, name?: string, uri: DocumentUri, ranges: Range[]}`

However, I've got a string. @blinry, is that a typo in the doc?